src/JobPanel/ui/main_window.py

```diff
@@ -446,7 +446,14 @@ def _handle_stop_multijob_action(self):
         self.frontend_service.mj_stop(key)
 
     def _handle_options(self):
-        OptionsDialog(self, self.data,self.data.env_presets).show()
+        prewFileName = self.data.workspaces.get_path()
+        dialog = WorkspaceSelectorWidget(self, self.data.workspaces.get_path())
+        dialog.select_workspace()
+        if prewFileName != dialog.value:
+            self.data.reload_workspace(dialog.value)
+            if not Analysis.exists(self.data.workspaces.get_path(), self.data.config.analysis):
+                self.data.config.analysis = None
+            self.data.config.save()
 
```

Comment:
Tady by se asi mělo zavřít hlavni okno, jako se to dělo v OptionsDialog pomocí self.parent().close().